etc/hooks/pre-commit.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+FILES=$(git diff --cached --name-only --diff-filter=ACMR | sed 's| |\\ |g')
```

Comment:
could you comment what this sed does

Reply:
Added a comment. Its auto escaping any files with spaces in it